src/IB/ConstraintIBMethod.cpp

```diff
@@ -217,6 +217,8 @@ ConstraintIBMethod::ConstraintIBMethod(std::string object_name,
       d_vel_com_def_new(d_no_structures, std::vector<double>(3, 0.0)),
       d_omega_com_def_current(d_no_structures, std::vector<double>(3, 0.0)),
       d_omega_com_def_new(d_no_structures, std::vector<double>(3, 0.0)),
+      d_center_of_mass_unshifted_current(d_no_structures, std::vector<double>(3, 0.0)),
+      d_center_of_mass_unshifted_new(d_no_structures, std::vector<double>(3, 0.0)),
```

Comment:
These need to be after `d_center_of_mass_new` to match the declaration order in the header.